tensorflow/python/ops/random_ops.py

```diff
@@ -310,21 +310,29 @@ def random_crop(value, size, seed=None, name=None):
   # If size > value.shape, zero pad the result so that it always has shape
   # exactly size.
   with ops.name_scope(name, "random_crop", [value, size]) as name:
+    size_dim = size
     value = ops.convert_to_tensor(value, name="value")
     size = ops.convert_to_tensor(size, dtype=dtypes.int32, name="size")
     shape = array_ops.shape(value)
-    check = control_flow_ops.Assert(
```

Comment:
Is the check no longer necessary?

Reply:
Thanks @alextp. Since the check is considering when the situation `size > value.shape` , if the edge case was accepted, the check may no longer be needed. May I have your suggestions?